templates/lune/lune.amp.html

```diff
@@ -18,6 +18,69 @@
   {{> ../../utils/amp-page-head.snip.html}}
   <body class="lune">
     {{> ../../components/navbar/navbar.snip.html}}
+
+    <!-- Used to achieve the clipping effect when scrolling the page -->
+    <div class="hide">
+      <svg width="0" height="0">
+          <defs>
+            <clipPath id="svgPath" clipPathUnits="objectBoundingBox">
+              <rect y="0" x="0" width="1" height="0.99"/>
+            </clipPath>
+          </defs>
+      </svg>
+    </div>
+
+    <!-- Landing page -->
+    <div class="lune-page lune-page-landing">
+      {{#landing-sections}}
+        <!-- Landing section -->
+        <section class="lune-landing-section {{classes}}">
+          <div class="lune-landing-wrapper absolute left-0 bottom-0 right-0 flex flex-column items-center justify-center">
+            <!-- Used by screen reader / document outline, not visible -->
+            <h2 class="hide">{{full_title}}</h2>
+            <div class="lune-landing-content relative flex items-center justify-center flex-auto mb4">
+              <div class="lune-landing-image-wrapper relative col-12">
+                <!-- Chair image -->
+                {{#product_image}}
+                  <!-- WebP image -->
+                  <amp-img
+                    class="lune-anim-scale-up z1"
+                    alt="{{alt}}"
+                    srcset="{{webp_srcset}}"
+                    width="{{width}}"
+                    height="{{height}}"
+                    layout="responsive">
+                      <!-- PNG Fallback -->
+                      <amp-img
```

Comment:
This fallback is intended for browsers that do not support webp, so an SVG would not be suitable in this case. We're following the example found [on this page](https://www.ampproject.org/docs/reference/components/amp-img) Go to the "Example: Specifying a fallback image" section